appendices/symbol-lookup-cheatsheet.md

```diff
@@ -41,7 +42,7 @@ Here is a more elaborate explanation of Pony's use of special characters:
 | { }  | a function type
 | :  | (1) after a variable: is followed by the type name 
      | (2) to indicate a function return type
-|    | (3) a type constraint
+     | (3) a type constraint
```

Comment:
The mixing of use of leading and non-leading | as option strikes me as error prone.

Any reason not to have the leading | for all entries?